### DISCUSSION TITLE
Category: Albuns

### DISCUSSION THREAD

**Discussion by rubengr** [photoprism/photoprism#5139] [Ideas]
As a user I want to organize photos/videos by folder when creating albuns.

At the moment PhotoPrism can create albuns but doens't organize it as folders. Could be a better organization method if user can organize photos in folders when add it to albuns.

**Comment by graciousgrey** [comment]
Thank you for your feedback! 

>At the moment PhotoPrism can create albuns but doens't organize it as folders.

Can you describe in more detail what you mean by this? Do you want PhotoPrism to move files in the file system when you add them to albums?

**Reply by rubengr** [reply]
Exactly! Folders according albuns
